app/Http/Controllers/ModulController.php

```diff
@@ -128,4 +130,23 @@ public function destroy($id)
 
         return '{"message": "success"}';
     }
+
+    public function updateJawabanConfiguration(Request $request){
+        $request->validate([
+            'id'         => 'required',
+            'idINT'         => 'required',
```

Comment:
Okay, the international class as usual needs special treatment, but what is the default value for this if we are handling regular class?

Reply:
Since that value is required

Reply:
I set `idINT` default as `null`. Since the toggle will lock/unlock both REG and INT module, I set `idINT` value with switch-case in Jawaban.vue script like:
```
switch (modul.id) {
        case 1:            // id of first module in REG
          modul.idINT = 6; // id of first module in INT
          break;
        case 2:
          modul.idINT = 7;
          break;
           ...
           ...
           ...
        default:
          modul.idINT = null;
          break;
      }

Reply:
Seems good from my side, resolving